modules/ROOT/pages/content-behavior-options.adoc

```diff
@@ -11,6 +11,8 @@ include::partial$configuration/draggable_modal.adoc[]
 
 include::partial$configuration/keep_styles.adoc[]
 
+include::partial$configuration/editable_class.adoc[]
```

Comment:
I intentionally moved this as the links to these sections were different and it shouldn't have been. As such it was a bug that they were in different pages.